inginious/frontend/pages/utils.py

```diff
@@ -133,6 +133,23 @@ def logger(self) -> logging.Logger:
         """ Logger """
         return logging.getLogger('inginious.webapp.pages')
 
+    def is_tos_defined(self):
```

Comment:
You shouldn't need this if you rely on the same mechanisms as for ``welcome_page``